chef/cookbooks/provisioner/recipes/setup_base_images.rb

```diff
@@ -60,8 +63,9 @@
 end
 
 # PXE config
-%w(x86_64).each do |arch|
-  # Make it easy to totally disable/enable ppc64le
+# ppc64le bootloader can parse pxelinux config files
+%w(x86_64 ppc64le).each do |arch|
```

Comment:
Because this bit is only for PXE boot, we have another loop later on for EFI boot. So we would need to do the filter twice.